samples/nrf9160/lwm2m_client/src/events/location_event_handler.c

```diff
@@ -9,7 +9,7 @@
 
 #include <zephyr/net/lwm2m_path.h>
 #include <net/lwm2m_client_utils_location.h>
-#include <net/lwm2m_client_utils_location_events.h>
+#include <location_events.h>
```

Comment:
I think you should use double quotes instead when including `location_events.h` now, because it's part of the sample, not a library anymore.